tests/python/relay/aot/aot_test_utils.py

```diff
@@ -303,40 +328,34 @@ def emit_main_micro_include(main_file, mod_name):
     main_file.write(f"#include <{mangle_module_name(mod_name)}.h>\n")
 
 
-def create_main(test_name, input_map, output_list_map, output_path, interface_api, workspace_bytes):
+def create_main(test_name, networks, output_path, interface_api, workspace_bytes):
```

Comment:
nit: i think we generally have preferred `models` over `networks`

Reply:
Sorted :+1: